astyanax-queue/src/main/java/com/netflix/astyanax/recipes/queue/ShardedDistributedMessageQueue.java

```diff
@@ -148,11 +149,11 @@ public class ShardedDistributedMessageQueue implements MessageQueue {
     final static AnnotatedCompositeSerializer<MessageMetadataEntry>   metadataSerializer
         = new AnnotatedCompositeSerializer<MessageMetadataEntry>(MessageMetadataEntry.class);
 
+    @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "type")
```

Comment:
please provide more context on this change

Reply:
I found this solution on [StackOverflow](https://stackoverflow.com/questions/58302380/how-to-i-disable-defaulttyping-is-it-disabled-by-default). But, I indicated the “type” property instead of the “className” because I applied it to the local variable